VideoWeb/VideoWeb/ClientApp/src/app/services/api/video-web.service.ts

```diff
@@ -91,7 +91,7 @@ export class VideoWebService implements IVideoWebApiService {
     }
 
     getVenues(): Observable<HearingVenueResponse[]> {
-        return this.apiClient.getVenues();
+        return this.apiClient.getVenues_();
```

Comment:
is this a typo?

Reply:
@omio-ma 
 
public async Task<ActionResult<IList<HearingVenueResponse>>> GetVenues()

That's swagger generated which we don't have much control.

Reply:
But it shouldn't have the underscore after it right?

Reply:
turned out to be a typo in the operation name, its resolved now